src/test/java/duke/ParserTest.java

```diff
@@ -0,0 +1,39 @@
+package duke;
+
+
+import org.junit.jupiter.api.Test;
+
```

Comment:
Perhaps you could potentially group the two imports statements into the same paragraph?